code/modules/cm_preds/falcon.dm

```diff
@@ -0,0 +1,108 @@
+/obj/item/falcon_drone
+	name = "falcon drone"
+	desc = "An agile drone used by Yautja to survey the hunting grounds."
+	icon = 'icons/obj/items/hunter/pred_gear.dmi'
+	icon_state = "falcon_drone"
+	item_icons = list(
+		WEAR_L_EAR = 'icons/mob/humans/onmob/hunter/pred_gear.dmi',
+		WEAR_R_EAR = 'icons/mob/humans/onmob/hunter/pred_gear.dmi',
+		WEAR_L_HAND = 'icons/mob/humans/onmob/hunter/items_lefthand.dmi',
+		WEAR_R_HAND = 'icons/mob/humans/onmob/hunter/items_righthand.dmi'
+	)
+	flags_equip_slot = SLOT_EAR
+	flags_item = ITEM_PREDATOR
+
+/obj/item/falcon_drone/get_examine_location(var/mob/living/carbon/human/wearer, var/slot, var/t_he = "They", var/t_his = "their", var/t_him = "them", var/t_has = "have", var/t_is = "are")
+	switch(slot)
+		if(WEAR_L_EAR)
+			return "on [t_his] shoulder"
+		if(WEAR_R_EAR)
+			return "on [t_his] shoulder"
+	return ..()
+
+/obj/item/falcon_drone/verb/control_falcon_drone()
+	set name = "Control Falcon Drone"
+	set desc = "Activates your falcon drone."
+	set category = "Yautja.Misc"
+	set src in usr
```

Comment:
make this also trigger on use_hand

also slightly confusing UX that you use a verb to start using the drone and an action button to stop using it. Ideally both would use action buttons.

Reply:
added the in-hand, and eh. the action buttons aren't really needed for a once-in-a-while drone